proposals/privacy.md

```diff
@@ -166,6 +174,7 @@ if the data is protected by another layer of authentication and encryption, for
 carried on a private network (for example, an encrypted VLAN).
 
 ### Context files MUST NOT be dereferenced if these dereferences can be associated with specific Things
+ML: This is a requirement for a consumer that cannot be enforced.
```

Comment:
None of this can be enforced... "MUST" is still in the context of a W3C Recommendation, there is generally no compliance test for W3C specs.   But we can frown at people, maybe use harsh language occasionally...